src/badguy/badguy.hpp

```diff
@@ -107,7 +117,8 @@ class BadGuy : public MovingSprite,
   bool is_in_water() const;
 
   /** Get melting particle sprite filename */
-  virtual std::string get_water_sprite() const {
+  virtual std::string get_water_sprite() const
+  {
     return "images/objects/water_drop/water_drop.sprite";
   }
 
```

Comment:
I think you could put that one on one line. Does it look fair enough when on one line?

Reply:
Yeah
Fixed